protos/sources/vega/governance.proto

```diff
@@ -442,6 +446,12 @@ enum ProposalError {
   PROPOSAL_ERROR_SPOT_PRODUCT_DISABLED = 47;
   // Market proposal is invalid, either invalid insurance pool fraction, or it specifies a parent market that it can't succeed.
   PROPOSAL_ERROR_INVALID_SUCCESSOR_MARKET = 48;
+  // Governance transfer proposal is invalid
+  PROPOSAL_ERROR_GOVERNANCE_TRANSFER_PROPOSAL_FAILED = 49;
+  // Governance transfer proposal failed
+  PROPOSAL_ERROR_GOVERNANCE_TRANSFER_PROPOSAL_INVALID = 50;
+  // Governance cancel transfer proposal is invalid
```

Comment:
can you? ;)

Reply:
Sure, once I figure out what it means

Reply:
it means that the proposal for cancelling an active governance proposal was invalid. Effectively it can only mean that the ID is wrong, i.e not an ID of an active recurring governance transfer.